components/permit-holders/modals/EditUserInformationModal.tsx

```diff
@@ -58,7 +57,7 @@ export default function EditUserInformationModal({
   useEffect(() => {
     setFirstName(applicant.firstName);
     setLastName(applicant.lastName);
-    setDateOfBirth(formatDate(applicant.dateOfBirth));
+    setDateOfBirth(new Date(applicant.dateOfBirth).toISOString().substring(0, 10));
```

Comment:
Reverted to how it was originally, using the formatting function displays the following before selecting a date
![image](https://user-images.githubusercontent.com/51224641/133518385-7623012d-0443-4e11-a2fa-21709567d78d.png)

Instead of displaying the initial date of birth (before editing)
![image](https://user-images.githubusercontent.com/51224641/133518478-cb7fd605-49a7-47b1-aae4-454158477e28.png)

Reply:
is now ` const [dateOfBirth, setDateOfBirth] = useState(formatDate(new Date(), true));`. Passing true into the `formatDate` function executes `new Date(applicant.dateOfBirth).toISOString().substring(0, 10)`